docs/src/man/optional_dependencies.md

```diff
@@ -0,0 +1,115 @@
+# Optional Features
+
+This section summarizes each optional dependency and the features available when used together with `LazySets.jl`.
+Internally, optional dependencies in Julia are handled with the package [Requires.jl](https://github.com/JuliaPackaging/Requires.jl).
+
+```@contents
+Pages = ["optional_dependencies.md"]
+```
+
+## Installing all dependencies
+
+Use the following command to install *all* optional dependencies. Installing all optional dependencies is required
+if you want to run the full test suite and build the documentation locally. 
+
+```julia
+julia> ] add AbstractTrees
+             BenchmarkTools
+             CDDLib
+             Distributions
+             Documenter
+             Expokit
+             ExponentialUtilities
+             IntervalConstraintProgramming
+             IntervalMatrices
+             LaTeXStrings
+             Makie
+             MiniQhull
+             RecipesBase
+             Symbolics
+             Optim
+             Plots
+             Polyhedra
+             StaticArrays
+             TaylorModels
```

Comment:
```suggestion
julia> ] add CDDLib
             Distributions
             Documenter
             Expokit
             ExponentialUtilities
             IntervalConstraintProgramming
             IntervalMatrices
             Makie
             Optim
             Polyhedra
             RecipesBase
             StaticArrays
             Symbolics
             TaylorModels
```
This is the list of `[targets]` from `Project.toml` (without `GR`), which is what is installed in CI. The other packages are only needed to build the docs, but for that we have a separate `Project.toml` file.